app.js

```diff
@@ -29,6 +27,22 @@ const friendData = [
     },
 ];
 
+const defaultFriends = [
```

Comment:
Im not sure these default friends are coming through, in general with half bakeds you can leave any working code that doesn't have comments.